lib/init.g

```diff
@@ -248,10 +250,15 @@ CallAndInstallPostRestore( function()
 
     if IsHPCGAP then
       BindThreadLocal( "BreakOnError", not GAPInfo.CommandLineOptions.T );
```

Comment:
Not really relevant to this PR: We should comment here why these variables are thread local (I suppose we don't want a break look in a worker thread; I have had students trying to debug task-based code, which was a big pain becuase of no break loops or error messages.

Reply:
Did that too.